src/app/app.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
+  title = 'basic';
```

Comment:
You can change the title to `Blog App` or whatever you prefer.

Reply:
Changed bhai. Thanks bhai.